tests/cluster_testing.py

```diff
@@ -140,6 +141,145 @@ def print_report_summary (fullreport):
           (pass_cnt, pass_clr, fail_cnt, fail_clr))
 
 
+def create_selfsigned_cert(ssl, cn):
```

Comment:
I'm thinking these are generally useful and should be in trivup, can you open a PR there instead - I'll merge it quickly and make a new release?